src/OSPSuite.Infrastructure.Import/Core/AbstractImporterException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using OSPSuite.Utility.Exceptions;
+
+namespace OSPSuite.Infrastructure.Import.Core
+{
+   public abstract class AbstractImporterException : OSPSuiteException
```

Comment:
@abdelr @georgeDaskalakis I have renamed it to be SINGULAR (one exception). Also constructor on abstract class should be protected.